app/models/zillow.rb

```diff
@@ -19,22 +19,32 @@ def initialize(attrs)
     @type         = attrs["searchresults"]["response"]["results"]["result"]["useCode"]
     @neighborhood = attrs["searchresults"]["response"]["results"]["result"]["localRealEstate"]["region"]["name"]
     @sqft         = attrs["searchresults"]["response"]["results"]["result"]["finishedSqFt"]
+    @rent         = attrs["searchresults"]["response"]["results"]["result"]["rentZestimate"]
+    if attrs["searchresults"]["response"]["results"]["result"]["lastSoldPrice"]["__content__"]
```

Comment:
This has the same problem, since `["lastSoldPrice"]` is `nil`, calling `["__content__"]` on it gives an error, you want to leave `["__content__"]` off of the conditional.